lib/Hero/Carousel.js

```diff
@@ -1,63 +1,70 @@
-import React, { useState } from "react";
-import { AiOutlineLeft, AiOutlineRight } from "react-icons/ai";
-import Swipe from "react-easy-swipe";
+import React, { useState } from 'react';
+import leftArrow from '../../public/arrow left.svg';
```

Comment:
I would highly discourage using spaces in file names. That's a big no-no in my books.